### DISCUSSION TITLE
Can implements be turned off while turning?

### DISCUSSION THREAD

**Discussion by sjmills** [Courseplay/Courseplay_FS22#1176] [Q&A] [answered]
I’m using the John Deere HX20 batwing mower, and it doesn’t shut off when CP comes to the end of a row or performs a turn. It really should. CP takes so long to calculate the execute turns that it’s wasting lots of fuel by keeping the RPMs up to run the mower. This should probably be an option, since some tools take too long to spool up and/or don’t require high RPM PTO.

**Comment by Tensuko** [comment, marked as answer]
We won't shut down the tools for a turn.

**Comment by sjmills** [comment]
Can I ask why? Like I said, it's really fuel wasteful because it takes so long to turn around. If it's an option, then those of us that would like to do this can, and the rest can leave things running the whole time.

**Reply by Tensuko** [reply]
To much effort and low benefit. I also dont want another option to an already complex mod.

Create a statistic for fuel used in turn with mower on and mower off.
Keep in mind without load (mower raised) it is much less.

**Reply by sjmills** [reply]
I don't know how to look at fuel usage.

**Comment by pvaiko** [comment]
> CP takes so long to calculate the execute turns 

How long? What turns?

**Reply by sjmills** [reply]
I did some timing and recorded a video of the row end turn. On this fairly small field, completing 1 row takes 39 seconds from the point after it lowers the mower and starts accelerating to the point after it slows down and finally stops after completing the row. The time it takes to turn around to start the next row is 55 seconds. This video shows it coming up to the row end, slowing down a bunch (pretty early), then backing up so it can turn around, then going too far, then backing up, then finally starting the row. This demonstrates that it takes much longer to turn around than it does to complete the row. This is why shutting off the tool while turning would be a very good feature to have.

Turning off "turn on field" shortens the turnaround time to much less, but that's a moot point for when the fields have tight boundaries of trees or fences.

I'm not sure why it changes gears so much. It's on automatic.

https://www.dropbox.com/s/4au9edx5exycchx/TurningMowerAtRowEnd.mp4?dl=0

**Reply by Tensuko** [reply]
Thats a problem of your settings dude...
Turn on field off and your turn is a simple U turn.
Skip rows helps too...

**Reply by pvaiko** [reply]
And/or add more headlands. 

The only thing we can improve on here is the going too far in the new row and probably those stops when backing up...

The rest is geometry.

**Reply by sjmills** [reply]
Like I said, turning "turn on field" off will help here, but not on fields that have objects too close.

If you can improve the 2 things pvaiko said, that would help everyone.

And still, a simple option to turn off tools when turning would be great to have.

**Reply by Tensuko** [reply]
> And still, a simple option to turn off tools when turning would be great to have.

As I said that won't happen, in RL they don't do it as well.

More Headlands is a simple solution and skip rows helps too.

Change "too far in the row" might cause troubles on plows or other towed implements not allignt correctly. Wich is already on the edge for plows.

Edit:
In addition, lifted tools do not create a high load on the vehicle wich should result in lower fuel use anyway.
